### ISSUE TITLE
[FEAT] Complete Redesign of Vendor's UI

### ISSUE THREAD

**Issue by Vimall03** [Vimall03/Alimento#44]
### Description:
The **vendor's section** of the website requires a **complete UI redesign** to improve appearance, usability, and consistency across all vendor-related pages. There are approximately **four pages** that need to be redesigned, not just the homepage. This is a **level 3 issue**, so please be prepared to fully redesign the UI for all vendor pages.

Contributors are required to first submit a **design proposal (e.g., via Figma)** for review. Based on the design, you will be assigned to proceed with development. The redesign should focus on improving layout, typography, color schemes, and overall user experience.

### Pages to be Redesigned:
1. Vendor Homepage
2. Vendor Dashboard
3. Vendor Restaurant Management Page
4. Vendor Settings Page

### Requirements:
- Apply the redesign to all vendor-related pages.
- Ensure consistency in design across the pages.
- The UI should be clean, user-friendly, and visually appealing.
- You may adjust layout, font size, color schemes, and other stylistic elements to improve the user experience.
- You can use **Bootstrap**, **Tailwind CSS**, or any other frameworks, but ensure all links are functional and no broken links remain.

### Steps to Reproduce:
1. Clone the project and set it up locally. (Refer to the setup section in the README)
2. Navigate to the vendor's login page: `http://localhost/alimento/vendor/vendor_login.php`
3. Use the following credentials to access the vendor page:
   - **Email:** aftereditofficial@gmail.com
   - **Password:** Test@1234
4. Review the current UI across all vendor-related pages and identify areas for improvement.

### Screenshot:
![image](https://github.com/user-attachments/assets/ec94be10-6f3e-4786-a085-3676b62ca565)

### Files Involved:
The relevant files for the vendor’s pages are located in:
`alimento/vendor`

### Notes:
- This is a **level 3 issue**, which requires a complete redesign of all vendor-related pages.
- Before you start, **please share a design** (e.g., via Figma) for approval. Once the design is reviewed and approved, you will be assigned the issue.
- If you'd like to create your own credentials or add a new restaurant, you can sign up via the vendor registration page or use the provided credentials to log in and test the UI changes.
- Feel free to suggest any additional UI/UX improvements.

**Comment by Sourabh782** [comment]
hii @Vimall03 can i work on it?

**Comment by Vimall03** [comment]
Hi @Sourabh782 ,

Once you've set up the project on your local machine, you can access the vendor's page here: `http://localhost/alimento/vendor/vendor_login.php.`

The vendor's page functions as an admin panel where each vendor or restaurant can manage their orders, update statuses, and edit menu details. As shown in the screenshot provided, the current design is quite basic, with just a table and a generic button.

I would like you to completely redesign the vendor's dashboard along with the other pages of the vendor section. You have the creative freedom to make the design visually appealing, but **please ensure all current features—such as updating the menu and managing pending orders—continue to work as expected after the new design is implemented.**

**Comment by Sourabh782** [comment]
Ya I have basic knowledge of php, and I'll share ss @Vimall03 /video of working before pr

**Comment by Vimall03** [comment]
Cool, Get started @Sourabh782. All the best!

**Comment by Vimall03** [comment]
Hey @Sourabh782, Have you made any progress ?

**Comment by Sourabh782** [comment]
hii @Vimall03  yup i did some research on it will soon share ss, currently in clg

**Comment by Vimall03** [comment]
Okay, Take your time.

**Comment by Vimall03** [comment]
Hey @Sourabh782, Is the design ready ?

**Comment by Sourabh782** [comment]
hey Vimal really sorry was busy in some family stuff cause of festives, can i submit till tommorow?

**Comment by Sourabh782** [comment]
hey @Vimall03, i am really sorry for late responding, but i will share screen shots of final result till 3-4 pm some touches left, really sorry

**Comment by Sourabh782** [comment]
@Vimall03 can u check these:
![Screenshot 2024-10-15 161258](https://github.com/user-attachments/assets/594da36d-0e87-4cd2-8953-9eadbb1a8c27)
![Screenshot 2024-10-15 161243](https://github.com/user-attachments/assets/a8827703-2f01-4024-b8ce-8868414ad7cb)
![Screenshot 2024-10-15 161231](https://github.com/user-attachments/assets/361e8e9f-1e32-4446-a852-1a7398603200)
![Screenshot 2024-10-15 161220](https://github.com/user-attachments/assets/25e94d70-10e7-4a48-a250-9e4017c60c37)

**Comment by Sourabh782** [comment]
these are the basic layout, i'll make proper page of them, but till then please verify these basic layout, then ill start working on the page

**Comment by Sourabh782** [comment]
> @Sourabh782, Looking good

Okk @Vimall03, so I'll refine the designing part a bit and will be sending pr till evening 👍👍

**Comment by Vimall03** [comment]
Okay, cool. Try to change the navbar and add it to all the pages in the vendor's section.

**Comment by Sourabh782** [comment]
> Okay @Sourabh782 , cool. Try to change the navbar and add it to all the pages in the vendor's section.

Which navbar? That coloured part?

**Comment by Vimall03** [comment]
Yes, it looks like a banner right now. So, just add a simple navbar that consists of the brand name on the left and a logout button on the right
